### DISCUSSION TITLE
Bar by Week or Month

### DISCUSSION THREAD

**Discussion by UlrichThiess** [kalkih/mini-graph-card#1134] [Ideas]
I wish group_by has parameter like week or month.

**Comment by valimaties** [comment]
This will be indeed a great feature.
For example, I have a cumulative sensor which calculates the difference between the energy exported to grid and the energy consumed from grid. This is practically the energy which will be report at each month on the suplier bill. This sensor will reset to 0 at the beginning of each month. So, I need a bar chart to display the last value from each month. 
More than that, it would be great that on bar hover/click, the information displayed to be the value and the month / year. Something like:

**623,28 kWh
March, 2025**
